src/components/Header/style.ts

```diff
@@ -20,6 +20,10 @@ export const Inner = styled.div`
   }
 `;
 
+export const IconContainer = styled.div`
+  cursor: pointer;
+`;
```

Comment:
```suggestion
export const IconContainer = styled.button`
`;
```

button의 역할을 수행하고 있는 element로 확인됩니다

Reply:
맞습니다 다음부터는 element를 용도에 맞게 사용하도록 하겠습니다